### DISCUSSION TITLE
DBW Throttle Body air bleed

### DISCUSSION THREAD

**Discussion by DeeEmm** [DIY-Flow-Bench/DIY-Flow-Bench#150] [General]
So tonight I managed to get some spare time to make a quick experiment with the DBW Throttle Body that I purchased.

The unit I bought is a Bosch 0 280 750 202 which is the 70mm unit commonly found on Commodores here in Australia. (cheap and plentiful.) A similar unit is used on GM cars and Alfas in Europe.

![PXL_20230509_100222308 MP](https://github.com/DeeEmm/DIY-Flow-Bench/assets/3038710/8701ee09-fe60-40fa-9ed4-e28681542502)

The pin connections for the motor are pins 3 and 5. These are labelled inside the socket on the unit.

I had a PWM DC motor controller handy from another project. This can take both AC and DC supply and also includes a potentiometer for manual control of the output. 

![PXL_20230511_100037629 MP](https://github.com/DeeEmm/DIY-Flow-Bench/assets/3038710/4c5b0e8c-d1a0-4061-9592-6fb779840f40)

I hooked it up to 12v and took it for a test. Well okay maybe it was more like 15v in the video...

https://www.youtube.com/watch?v=2HU2Kh0YHfc

The position regulation is pretty good using the potentiometer and so I don't expect that there would be any issues controlling it from the shield. It also does not move easily so is not deflected out of position by the application of some small force to the butterfly. I would expect it to be pretty stable.

Next stage is really to get it installed in a closed loop setup and look at using it to regulate the test depression.

**Comment by DeeEmm** [comment]
Drafted a quick adaptor for the flanged side of the TB so that it can be tee'd into the pipework using a regular rubber coupling

<img width="2162" alt="Screenshot 2023-05-12 at 10 03 17 am" src="https://github.com/DeeEmm/DIY-Flow-Bench/assets/3038710/05dcd980-3806-4af5-80b3-912ecb17080b">

Here's the STL for anyone who wants to use it.

[70mm_DBW_TB_Adaptor.stl.txt](https://github.com/DeeEmm/DIY-Flow-Bench/files/11457226/70mm_DBW_TB_Adaptor.stl.txt)

There is also a version with a scolloped top top edge to suit 100mm OD pipe so that it can be glued directly to the pipework.

<img width="2160" alt="Screenshot 2023-05-12 at 10 19 09 am" src="https://github.com/DeeEmm/DIY-Flow-Bench/assets/3038710/1c718fc3-bacc-4cfb-8c30-6763518aeef6">

[70mm_DBW_TB_Adaptor_Profiled.stl.txt](https://github.com/DeeEmm/DIY-Flow-Bench/files/11457365/70mm_DBW_TB_Adaptor_Profiled.stl.txt)

Just change the file extension from TXT to whatever your 3D printer / slicer recognises

**Comment by black-top** [comment]
Looks great, I think it should work really well for pressure regulation.
